api/src/main/java/org/openmrs/api/impl/PatientServiceImpl.java

```diff
@@ -592,7 +587,22 @@ public void mergePatients(Patient preferred, Patient notPreferred) throws APIExc
 		personMergeLog.setPersonMergeLogData(mergedData);
 		Context.getPersonService().savePersonMergeLog(personMergeLog);
 	}
-	
+
+	private void requireNoActiveOrderOfSameType(Patient patient1, Patient patient2) {
+		List<Order> ordersByPatient1 = Context.getOrderService().getAllOrdersByPatient(patient1);
+		List<Order> ordersByPatient2 = Context.getOrderService().getAllOrdersByPatient(patient2);
+		ordersByPatient1.forEach((Order order1)->{
+			ordersByPatient2.forEach((Order order2)->{
+				if(order1.isActive() && order2.isActive() && order1.getOrderType().equals(order2.getOrderType())){
+					log.debug("Merge operation cancelled: Cannot merge patient" + patient1.getPatientId()
+						+ " to patient" + patient2.getPatientId());
+					log.debug("Patients having active order of " + order1.getOrderType());
+					throw new APIException("Patient.cannot.merge", (Object[]) null);
```

Comment:
Do I have to update the `"Patient.cannot.merge"` property just in the messages.properties file or all the *.properties files?
Some of those files have commented lines with `english text`.